src/twisted/conch/newsfragments/9315.feature

```diff
@@ -0,0 +1 @@
+twisted.conch.ssh.session.SSHSession can now accept environment variables sent by the client, using a new ISessionSetEnv interface.
```

Comment:
👍 Love the description.  Could you qualify `ISessionSetEnv` and perhaps be a little more specific than "using" though?  Like "by your SSH avatar implementing"?